backend/bird_ringing/settings.py

```diff
@@ -27,7 +27,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["localhost", "backend"]
```

Comment:
I think adding `backend` here is only needed if incoming requests come from the host `backend`?

Reply:
`backend` is needed for the compose healthcheck.